File: Technology Fundamentals/02.Exercises - Intro and Basic Syntax/Java/src/PadawanEquipment.java
import java.util.Scanner;

public class PadawanEquipment {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double amountMoney = Double.parseDouble(scanner.nextLine());
        int numberStudents = Integer.parseInt(scanner.nextLine());
        double numberLightsabers = Double.parseDouble(scanner.nextLine());
        double numberRobes = Double.parseDouble(scanner.nextLine());
        double numberBelts = Double.parseDouble(scanner.nextLine());

        double discountStudents = Math.ceil(numberStudents + (numberStudents * 0.10));
        double lightsabersPrice = numberLightsabers * discountStudents;
        double robesPrice = numberRobes * numberStudents;
        double discountBelts = numberStudents - numberStudents / 6;
        double beltsPrice = numberBelts * discountBelts;

        double price = lightsabersPrice + robesPrice + beltsPrice;

        if (price <= amountMoney)
        {
            System.out.printf("The money is enough - it would cost %.2flv.", price);
        }
        else
        {
            System.out.printf("Ivan Cho will need %.2flv more.",price - amountMoney);
        }
    }
}

File: C# Fundamentals May 2019/02.Exercises - Intro and Basic Syntax/PHP/PadawanEquipment.php
<?php
$amountMoney = floatval(readline());
$numberStudents = intval(readline());
$priceOfLightsabers = floatval(readline());
$priceOfRobes = floatval(readline());
$priceOfBelts = floatval(readline());

$discountStudents = ceil($numberStudents + ($numberStudents * 0.10));
$lightsabersPrice = $priceOfLightsabers * $discountStudents;
$robesPrice = $priceOfRobes * $numberStudents;
$discountBelts = ceil($numberStudents - $numberStudents / 6);
$beltsPrice = $priceOfBelts * $discountBelts;

$totalPrice = $lightsabersPrice + $robesPrice + $beltsPrice;

if ($totalPrice <= $amountMoney) {
    $totalPrice = number_format($totalPrice, 2, '.', '');
    echo "The money is enough - it would cost $totalPrice"."lv.";
} else {
    $needMoney = $totalPrice - $amountMoney;
    $needMoney = number_format($needMoney, 2, '.', '');
    echo "Ivan Cho will need $needMoney"."lv more.";
}

?>
